src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+    public static int calc(String str) {
+        if (isBlank(str)) {
+            return 0;
+        }
+
+        return sum(split(str));
+    }
+
+
+    private static String[] split(String str) {
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(str);
+        if (m.matches()) {
+            String customDelimiter = Pattern.quote(m.group(1));
+            String numbersPart = m.group(2);
+            return numbersPart.split(customDelimiter);
+        } else {
+            return str.split("[,:]");
```

Comment:
else 없이 return 하면 좋을 것 같아요 😃 
```suggestion
        }
        return str.split("[,:]");
```
```suggestion
        if (m.matches()) {
            String customDelimiter = Pattern.quote(m.group(1));
            String numbersPart = m.group(2);
            return numbersPart.split(customDelimiter);
        }
        return str.split("[,:]");
```